src/main.cpp

```diff
@@ -91,17 +86,17 @@ bool parse_config(char * config_file, server_options *server_options_pointer) {
 
   get_server_options(config, server_options_pointer);
   std::string port = server_options_pointer->port;
-  std::string static_file_root = server_options_pointer->static_file_root;
+  //std::string static_file_root = server_options_pointer->static_file_root;
 
   if (port == "") {
     std::cerr << "Please specify a port.\n";
     return false;
   }
```

Comment:
You're right, I was thinking more along the lines of making sure all of the mappings from `m_path` to `m_root` have a `m_root` that's set -- right now I think if the config file is set up incorrectly (i.e. something like `path /static1 StaticHandler{}` with no root specified), it'll still add it to the map and there'll be some undefined string for its root. Just error handling things -- everything looks good to me for the cases where the config file is formatted correctly